mlflow/store/sqlalchemy_store.py

```diff
@@ -68,6 +91,25 @@ def __init__(self, db_uri, default_artifact_root):
             with self.ManagedSessionMaker() as session:
                 self._create_default_experiment(session)
 
+    @staticmethod
+    def _initialize_tables(engine):
+        _logger.info("Creating initial MLflow database tables...")
+        LegacyBase.metadata.create_all(engine)
+        _upgrade_db(str(engine.url))
+
+    @staticmethod
+    def _verify_schema(engine):
+        mc = MigrationContext.configure(engine.connect())
+        diff = compare_metadata(mc, Base.metadata)
+        if len(diff) > 0:
+            _logger.error("Detected one or more differences between current database schema and "
+                          "desired schema, exiting. Diff:\n %s",
+                          pprint.pprint(diff, indent=2, width=20))
+            raise MlflowException(
+                "Detected out-of-date database schema. Run 'mlflow db upgrade %s' to migrate your "
```

Comment:
Yeah I actually like `mlflow db upgrade` because it's a bit more flexible, although maybe slightly harder to discover via `--help` since it's a nested command (i.e. you have to do `mlflow --help` to discover the `db` command group, then `mlflow db --help` & `mlflow db upgrade --help`, vs `mlflow --help` and `mlflow upgradedb --help`)

Reply:
Will make that change!